### PR TITLE
Feature 208 sosmeta translator

### PR DESCRIPTION
**Pull request by Jdallos**

### CODE REVIEW

File: src/utils/sosmeta-utils.ts [hunk 1]
```diff
@@ -153,6 +153,22 @@ namespace SosmetaUtils {
...
+   * @param sosmetaSection
+   * @returns MetaformField
+   */
+  const handleSosmetaBooleanSection = (sosmetaSection: any) => {
+    try {
+      const { required } = sosmetaSection;
+
+      return [ handleSosmetaBooleanField(sosmetaSection.sosmeta.name[0].value, sosmetaSection, required) ];
+    } catch (e) {
+      // TODO: Error handling/messages
```
Comment (EVP1989) [inline src/utils/sosmeta-utils.ts:168]: If trycatch is needed here, do proper error handling.

**Review by EVP1989 — APPROVED** [review]
Approved with comment.

File: src/utils/sosmeta-utils.ts (hunk 1 above)
Comment (Jdallos) [reply src/utils/sosmeta-utils.ts]: Try catch removed and found any errors were still being handled in the convertSosmetaToMetaform function.

**Review by EVP1989 — APPROVED** [review]
Approved

**Pull request merged**